mvrx/src/main/kotlin/com/airbnb/mvrx/DeliveryMode.kt

```diff
@@ -11,6 +11,7 @@ sealed class DeliveryMode {
         return when (this) {
             is RedeliverOnStart -> RedeliverOnStart
             is UniqueOnly -> UniqueOnly(properties.joinToString(",", prefix = subscriptionId + "_") { it.name })
+            is UniqueUpdatesOnly -> UniqueOnly(properties.joinToString(",", prefix = subscriptionId + "_") { it.name })
```

Comment:
@podarsmarty was this a copy paste mistake?
```suggestion
            is UniqueUpdatesOnly -> UniqueUpdatesOnly(properties.joinToString(",", prefix = subscriptionId + "_") { it.name })
```

Reply:
oops yea, my b